src/ExpressionEvaluator/CSharp/Test/ExpressionCompiler/TupleTests.cs

```diff
@@ -261,8 +261,8 @@ .maxstack  6
   IL_002f:  ldstr      ""y""
   IL_0034:  call       ""(int A, int B) Microsoft.VisualStudio.Debugger.Clr.IntrinsicMethods.GetVariableAddress<(int A, int B)>(string)""
   IL_0039:  ldloc.0
-  IL_003a:  stind.ref
-  IL_003b:  ret
+  IL_003a:  stobj      ""System.ValueTuple<int, int>""
```

Comment:
It is an error type because testcase does not reference System facade where ValueType lives. As a result ValueTuple was considered a class.
This code would not be emittable in regular compile scenario. It would have diagnostics about missing references.

---
In reply to: [81644864](https://github.com/dotnet/roslyn/pull/14161#discussion_r81644864) [](ancestors = 81644864,81644123)